scripting/weaponmodels.sp

```diff
@@ -233,7 +262,7 @@ public void OnPluginEnd()
 
 			if (viewModel1 != -1)
 			{
-				SetEntityVisibility(viewModel1, true);
```

Comment:
Check the engine version, use the frame delay function if g_iEngineVersion == Engine_SDK2013